nemascan-nxf.sh

```diff
@@ -9,6 +9,9 @@ DEFAULT_VCF_VERSION="20220216"
 DEFAULT_GOOGLE_PROJECT="andersen-lab"
 DEFAULT_GOOGLE_ZONE="us-central1-a"
 DEFAULT_GOOGLE_SERVICE_ACCOUNT_EMAIL="nscalc-201573431837@andersen-lab.iam.gserviceaccount.com"
```

Comment:
this code is not from us, but if possible, it shouldn't be exposing service accounts email addresses....